src/Agent.Worker/JobExtension.cs

```diff
@@ -66,7 +66,13 @@ public async Task<List<IStep>> InitializeJob(IExecutionContext jobContext, Pipel
 
                     // Set agent version variable.
                     context.Variables.Set(Constants.Variables.Agent.Version, BuildConstants.AgentPackage.Version);
+                    context.Output(StringUtil.Loc("AgentNameLog", context.Variables.Get(Constants.Variables.Agent.Name)));
+                    context.Output(StringUtil.Loc("AgentMachineNameLog", context.Variables.Get(Constants.Variables.Agent.MachineName)));
```

Comment:
would it also make sense to log the user the agent is running as?

Reply:
I've added a log that is "Agent running as: '_System.Environment.UserName_'"